client/src/locales/en.json

```diff
@@ -216,6 +216,10 @@
   "dashboard.index.no-arch-sacks": "There are no archived sacks yet.",
   "dashboard.index.no-sacks": "There are no sacks yet.",
   "dashboard.index.show-arch-sacks": "Show archived sacks",
+  "delete-link.has-expired": "Delete link has expired. Please go to {link} and find delete account option to get new delete link.",
```

Comment:
"Delete account link has expired. If you still want to delete account go to user profile page."